CvGameCoreDLL_Expansion2/CvPlot.cpp

```diff
@@ -15404,6 +15404,78 @@ int CvPlot::GetNumSpecificPlayerUnitsAdjacent(PlayerTypes ePlayer, const CvUnit*
 ///-------------------------------------
 
 #if defined(MOD_BALANCE_CORE)
+static int GetDefensiveApproachMultiplier(CivApproachTypes eApproach)
```

Comment:
Just to clarify, previosly eStrength and eApproach had initial value of 1, shouldn't there be a default case with return 1?

Reply:
So you won't fall through cases and NULL